Formula/eg-examples.rb

```diff
@@ -0,0 +1,38 @@
+class EgExamples < Formula
+  include Language::Python::Virtualenv
+
+  desc "Useful examples at the command-line"
+  homepage "https://github.com/srsudar/eg"
+  url "https://files.pythonhosted.org/packages/a6/93/38075713a7968a9e8484e894f604f99a68e443e0f9db0ed48063b1241969/eg-1.1.1.tar.gz"
+  sha256 "3faa5fb453d8ba113975a1f31e37ace94867539ba9d46a40af4cea90028a04e4"
+  head "https://github.com/srsudar/eg.git"
```

Comment:
Please remove the head spec.

https://github.com/Homebrew/homebrew-core/pull/33007#issuecomment-430048504

Reply:
Sure, will do so. I thought Formulae _could_ have a head spec, as they "_should not_ have a HEAD or devel spec" ‒ Is it _must not_ instead of _should not_? Anyway, I'll remove it.

Also, I thought ‒ from previous contributions ‒ that commits are _always_ squashed.